### DISCUSSION TITLE
Options to compare across folders

### DISCUSSION THREAD

**Discussion by OlloOndra** [qarmin/czkawka#1313] [Ideas]
Current options when selecting folders are not granular enough, as they mostly allow an user to select included and excluded folders.

They don't let you - for example - check if folder B has duplicates of folder A, without looking for additional duplicates WITHIN folder A or B.

When checking content, or working on very large sets of data, this can make the entire process a lot slower than necessary.

Could you consider adding options such as the below?

- Compare files across all folders (default behaviour)
- Compare files across different folders only
- Compare files within same folder only

Thank you

**Comment by ledditorandproud** [comment]
This would be super helpful. I posted a feature request for that third bullet point in particular a while ago. 

I have pictures categorised by subjects and themes (using folders), and while I want to remove duplicates from within a single category, I usually need to have one picture in multiple categories. As it is, I have to manually add and remove each and every folder I want to check. But with this feature, I'd be able to just select the entire collection, select option three, and have the entire process done in one go. 

It's the only issue I've ever had in years of using Czkawka, so I'd love to see this added someday.
